torch/onnx/_internal/exporters/fx_base.py

```diff
@@ -0,0 +1,41 @@
+import abc
+from typing import Callable, Mapping
+
+import torch._ops
+from torch.onnx._internal import _beartype
+from torch.onnx._internal.exporter import Exporter, ExportOutput
+from torch.onnx._internal.fx import function_dispatcher, passes
+
+# TODO: make_fx lose stack info https://github.com/pytorch/pytorch/issues/90276
+
+
+class FXGraphModuleExporter(Exporter, abc.ABC):
```

Comment:
Same as above (new stuff not discussed on v1 design proposal which deviates from the complete design)